django/people/consumers.py

```diff
@@ -0,0 +1,22 @@
+from asgiref.sync import async_to_sync
+from channels.generic.websocket import JsonWebsocketConsumer
+from people.serializers import PersonSerializer
+from people.models import Person
+
+class PersonConsumer(JsonWebsocketConsumer):
+    activation_code = None
```

Comment:
what is this for ?

Reply:
I guess is just leftover code from the consumer from where I got inspiration :D deleting.